change/@fluentui-react-table-bb208d47-ce79-49d3-bb79-3b635f0d5e95.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28161&commitId=d4315e0d0d5c3652b62feacb6d003d4512e59fab&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1941 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - truncate.default (disabled).chromium.png|11830| Changed|
|Table layout flex - truncate.false.chromium.png|11830| Changed|
|Table layout flex - truncate.true.chromium.png|6919| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>